docs/project/2024/Ioana_maria.cristea/index.md

```diff
@@ -0,0 +1,103 @@
+
+## Architecture 
+The **Raspberry Pi Pico** serves as the central control unit, directing and managing all other components utilized within the project.
+
+
+The **LCD** serves as an interface for showcasing the sensor readings, it is connected to the Pico through I2C
+
+The **servomotor** is connected to an improvised arm, allowing the fan to rotate 90° in either direction.
+
+The **buzzer**, when the temperature threshold is surpassed, emits an  alarm, acting as an alert mechanism.
+
+The **pressure and temperature** sensor detects changes in temperature and pressure, supplying input to the Pico.
+
+A **DC Motors** that controls the fan
+
+![diagram](proiectma.jpg)
+
+
+
+## Log
+
+<!-- write every week your progress here -->
+
+### Week 6 - 12 May
+
+### Week 7 - 19 May
+
+### Week 20 - 26 May
+
+## Hardware
+
+Detail in a few words the hardware used.
+
+### Schematics
+
+Place your KiCAD schematics here.
+
+### Bill of Materials
+
+<!-- Fill out this table with all the hardware components that you might need.
+
+The format is 
+```
+| [Device](link://to/device) | This is used ... | [price](link://to/store) |
+
+```
+
+-->
+
+| Device | Usage | Price |
+|--------|--------|-------|
+| [Rapspberry Pi Pico W](https://www.raspberrypi.com/documentation/microcontrollers/raspberry-pi-pico.html) | The microcontroller | [35 RON](https://www.optimusdigital.ro/en/raspberry-pi-boards/12394-raspberry-pi-pico-w.html) |
+| [BMP280](https://www.bosch-sensortec.com/media/boschsensortec/downloads/datasheets/bst-bmp280-ds001.pdf) | Temperature Sensor| [8,49 RON](https://www.optimusdigital.ro/ro/senzori-senzori-de-presiune/1666-modul-senzor-de-presiune-barometric-bmp280.html?search_query=bmp280&results=11) |
+| [Active Buzzer]() | Active Buzzer | [1,49 RON](https://www.optimusdigital.ro/ro/audio-buzzere/635-buzzer-activ-de-3-v.html?search_query=buzzer+activ&results=18) |
+| [LCD Display](http://www.lcdwiki.com/res/MSP1804/1.8inch_SPI_Module_MSP1804_User_Manual_EN.pdf) | LCD Display| [30 RON](https://www.optimusdigital.ro/ro/optoelectronice-lcd-uri/12392-modul-lcd-de-18-cu-spi-i-controller-st7735-128x160-px.html?search_query=lcd+st773&results=8) |
+| [DC Motor]() | DC Motor| [3,50 RON](https://www.emag.ro/motor-electric-dc-3-6v-ajmaker-8000rpm-6v-450-ma-25-x-20-x15-mm-gri-z000687/pd/DSFQLHYBM/?cmpid=93116&utm_source=google&utm_medium=cpc&utm_campaign=(RO:eMAG!)_3P_NO_SALES_%3e_Jucarii_hobby&utm_content=111476631565&gad_source=1&gclid=CjwKCAjw88yxBhBWEiwA7cm6pRCxklxkpkhCiPunMVr4_dPU2-iwMSfpSF-6S8OcYMyXWCNHiOyHPxoCQGIQAvD_BwE) |
+
+
+
+## Software
+
+| Library | Description | Usage |
+|---------|-------------|-------|
+| [st7789](https://github.com/almindor/st7789) | Display driver for ST7789 | Used for the display for the Pico Explorer Base |
+| [embedded-graphics](https://github.com/embedded-graphics/embedded-graphics) | 2D graphics library | Used for drawing to the display |
```

Comment:
Do you need these libraries?

Reply:
Hi, no sorry I thought that we had to change those for when we have the software milestone
I'll delete them for now.